doc: update readme to include 6.5.2 vcredist (#439)
cpeditor/cpeditor: README.md

- I am using Java and the editor can't run my codes.
   - You have to use a **non-public** class for your solution, its name can be configured in Preferences->Language->Commands->Java Class Name.
- I get **DLL Missing error** when launching the application?

>    - Please download  [Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads). The [6.5.2](https://github.com/cpeditor/cpeditor/releases/tag/6.5.2) bundles it with the installer.

Reviewer: ```suggestion
   - Please download  [Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads). Starting [6.5.2](https://github.com/cpeditor/cpeditor/releases/tag/6.5.2) we started bundling it with the installer.
```

>    - Please download  [Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads). Or you can install versions higher than or equal to [6.5.2](https://github.com/cpeditor/cpeditor/releases/tag/6.5.2) as since this version we bundle it with the installer.

Reviewer 2: ```suggestion
   - Please download  [Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads). Or you can install versions higher than or equal to [6.5.2](https://github.com/cpeditor/cpeditor/releases/tag/6.5.2) because since this version the VC Redist is bundled with the installer.
```
